### ISSUE TITLE
I want train my own speaker recognition system ,but i can't find the train code

### ISSUE THREAD

**Issue by wuqiangch** [Livefull/SphereDiar#1]
@Livefull Hi,I am sorry about your rejected journal article. I think the method in the paper is very meaningfull.I want train my own speaker recognition system ,but i can't find the train code.
In your paper "Combining LSTM, NetVLAD and Additive Margin
Softmax for Text-Independent Speaker Verification",https://github.com/Livefull/LSTM-based-speaker-verification.    But I can't find it.
Thanks!

**Comment by Livefull** [comment]
Hi!


I apologize for the late reply. In addition, I must mention that I have recently switched fields and currently work on medicine in HUS, Helsinki. Thus, I do not have too much time for speaker verification and also due to the coronavirus-situation lost access to most of my scripts. However, here are two for you to hopefully get you going: AM_emb_models.py and AM_training.py


AM_training.py describes how the additive margin softmax training is handled and the other one has the model configurations. I do not have the data generation functions but see the embed.py in the github to get the basic idea. I hope this helps you!


Best, Tuomas Kaseva

________________________________

**Comment by wuqiangch** [comment]
@Livefull Thanks for your reply. But can you  upload the two files(AM_emb_models.py and AM_training.py) in this project?
